src/common/snippets/include/snippets/lowered/loop_info.hpp

```diff
@@ -106,34 +265,176 @@ class LoopInfo {
      * @param T - transformation
      * @param args - arguments of the transformation
      */
-    template <SpecificIterationHandlers::HandlerType Type, typename T, class... Args>
-    void register_handler(Args&&... args) {
-        m_handlers.register_handler<Type, T>(args...);
+    template <SpecificLoopIterType Type, typename T, class... Args>
+    void register_pass_to_handler(Args&&... args) {
+        m_handlers.register_pass<Type, T>(args...);
     }
 
     /**
-     * @brief Update the parameters of existing loop input ports
-     * @param updater - function that updates ports
+     * @brief Sort ALL input Loop Ports by `new_order`: `m_input_ports[new_order[i]] = m_input_ports[i]`
+     * @param new_order vector of new indexes
+     */
+    void sort_input_ports(const std::vector<size_t>& new_order);
```

Comment:
If it takes `new_order` it's not really sorting (it takes bool ascending, and may be comparator).
Rename to `reorder_input_ports`?